index.html

```diff
@@ -2020,9 +2020,12 @@ <h3>Presentations</h3>
           <dd>
 The <a>verifiable presentation</a> MAY include a <code>proof</code>
 <a>property</a>. If present, the value SHOULD be used to express a securing
-mechanism such as [[?VC-DATA-INTEGRITY]]. A <a>verifiable presentation</a> MAY
-be secured using an external proof such as [[?VC-JWT]]. For details related to
-the use of this property, see Section <a href="#securing-verifiable-credentials"></a>.
+mechanism, such as [[?VC-DATA-INTEGRITY]], that is listed in the
+<a href="https://w3c.github.io/vc-specs-dir/#proof">Proof section</a> of the
+Verifiable Credentials Specifications Directory [[?VC-SPECS]]. A
+<a>verifiable presentation</a> MAY be secured using an <a>external proof</a> such as
+[[?VC-JWT]]. For details related to the use of this property, see Section
+<a href="#securing-verifiable-credentials"></a>.
```

Comment:
```suggestion
mechanism, such as those registered in the
Verifiable Credentials Specifications Directory [[?VC-SPECS]].
```

Reply:
If we take this change request, @OR13 -- none of the VC-COSE-JOSE securing mechanisms will be listed due to:

https://github.com/w3c/vc-specs-dir/pull/14 being blocked.

Only the VC Data Integrity mechanisms will be listed. 

Given the opposition we've traditionally had to favoring one mechanism over the other (your change request would favor Data Integrity over VC-COSE-JOSE) I suggest we don't take this change now until https://github.com/w3c/vc-specs-dir/pull/14 is unblocked, or we're listing all securing mechanism in the vc-specs-dir.

Reply:
@Sakurann ^ seems you are block the dependency, and I am blocking this PR, based on your block.. can we resolve?

Reply:
@msporny I hear what you are saying, but lets unblock the stuck PR .... by closing it, or merging it... before we take this change... its been open since April 8th... it should have been closed or merged by now.

Reply:
https://github.com/w3c/vc-specs-dir/pull/14 is unblocked.